libs/data-mapper/src/lib/models/Function.ts

```diff
@@ -26,6 +26,7 @@ export interface FunctionInput {
   allowedTypes: NormalizedDataType[];
   isOptional: boolean;
   allowCustomInput: boolean;
+  displayAttribute?: DisplayAttribute;
```

Comment:
Kind of an awkward name? Why not something simple like "inputType"?

Reply:
I want it to be distinct from "allowedTypes" but I'll find something else to call it

Reply:
That works

Reply:
done